src/index.js

```diff
@@ -0,0 +1,74 @@
+import './style.css';
+import List, { editTask, cleanAll } from './crud.js';
+import state from './interactive.js';
+
+const add = document.getElementById('send');
+const inputAdd = document.getElementById('add');
+const form = document.getElementById('form');
+const cleanBtn = document.getElementById('clean');
+
+const task = JSON.parse(localStorage.getItem('tasks')) || [];
+const loader = () => {
```

Comment:
OPTIONAL
The code structure could be improved by separating the loader() function into a separate module, which can be imported into the main file. This will help in better code organization and modularity.